modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -241,6 +242,7 @@ class CV_EXPORTS_W CascadeClassifier
     traincascade application.
      */
     CV_WRAP bool load( const String& filename );
+    CV_WRAP bool load( FileStorage& filename );
```

Comment:
Could you please remove CV_WRAP here? Precommit build is failed due to inability to provide python bindings for FileStorage object.